core/src/main/java/com/kin/ecosystem/core/data/settings/SettingsDataSource.kt

```diff
@@ -6,6 +6,10 @@ interface SettingsDataSource {
 
     fun isBackedUp(publicAddress: String): Boolean
 
+    fun hasSeenTransfer(kinUserId: String): Boolean
```

Comment:
add `Flow`postfix
can you change `isSaw` to `hasSeen` also ?

Reply:
done